pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/BaseClusterIntegrationTest.java

```diff
@@ -506,6 +506,13 @@ protected TableConfig getRealtimeTableConfig() {
     return getRealtimeTableConfig(getTableName());
   }
 
+  /**
+   * Returns the headers to be used for Pinot client transport.
+   */
+  protected Map<String, String> getPinotClientTransportHeaders() {
```

Comment:
nit: clearly specify what Pinot client transport means. I think it means the JDBC Connection `PinotConnection`? You can perhaps do that link thing: `{@link PinotConnection}`

Similar comment for connectionClientRequest and segmentUploadAuth methods.